src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Views/UnoQuickStart.csproj

```diff
@@ -43,7 +43,8 @@
 		<ApplicationDefinition Include="App.xaml" Condition="exists('App.xaml')" />
 		<Compile Update="**\*.xaml.cs">
 			<DependentUpon>%(Filename)</DependentUpon>
-		</Compile>
+		</Compile>		
+		<PriResource Include="**\*.resw" />
```

Comment:
We just removed this from the extensions template as it looks like resw files are included by default as Priresource (feedback was that it was generating duplicates). Just confirming that we should revert this and include them?

Reply:
`PRIResources` in extensions where unconditional (so duplicated on windows), this one is conditional to not windows. I missed that part on extensions as well, my apologies :)